src/main/entity/User.ts

```diff
@@ -14,20 +13,25 @@ export class User {
     lastName: string,
     email: string,
     username: string,
-    password: string
+    idFirebase: string
```

Comment:
What do you think about:
```suggestion
    firebaseId: string
```

Reply:
I refactored!

Reply:
I think you should change it to `firebaseId` in this function too!